tuxemon/event/actions/spawn_monster.py

```diff
@@ -58,15 +60,15 @@ class SpawnMonsterAction(EventAction[SpawnMonsterActionParameters]):
 
     def start(self) -> None:
         npc_slug, breeding_mother, breeding_father = self.parameters
-        world = self.session.client.get_state_by_name("WorldState")
+        world = self.session.client.get_state_by_name(WorldState)
         if not world:
-            return False
+            return
 
         npc_slug = npc_slug.replace("player", "npc_red")
         trainer = world.get_entity(npc_slug)
         if trainer is None:
             logger.error(f"Could not find NPC corresponding to slug {npc_slug}")
-            return False
+            return
```

Comment:
Wait, I thought returning true or false was how we knew if the event action failed?

Reply:
I didn't see the return value of that method used anywhere. Do you know where is it checked?

Reply:
no i don't think it was ever like that.  scripts don't have flow control, and every action is executed in order until the end, regardless of the status of another.